pt-BR/association_basics.md

```diff
@@ -678,7 +678,7 @@ module MyApplication
 end
 ```
 
-This will work fine, because both the `Supplier` and the `Account` class are defined within the same scope. But the following will _not_ work, because `Supplier` and `Account` are defined in different scopes:
+Isto funciona corretamente, porque tanto a classe `Supplier` quanto a classe `Account` são definidas dentro do mesmo escopo. Mas o próximo exemplo _não_ funcionará, porque `Supplier` e `Account` são definidos em escopos diferentes:
```

Comment:
Acho que como usamos muito itálico esse `_não_` talvez pudesse virar negrito. Mas confesso que acho que não faria muita diferença então é mais um questionamento para o futuro :thinking:

Reply:
Concordo.
Acho que resolvendo esse comentário já podemos fechar esse PR. :)

Reply:
Só conferindo se eu entendi certo: podemos transformar o `_não_` aqui em negrito?

Reply:
Acho que ficaria legal, mas seria mas um polimento. Vou deixar um suggestion e qualquer coisa você faz o merge aqui pelo GitHub mesmo.

```suggestion
Isto funciona corretamente, porque tanto a classe `Supplier` quanto a classe `Account` são definidas dentro do mesmo escopo. Mas o próximo exemplo __não__ funcionará, porque `Supplier` e `Account` são definidos em escopos diferentes:
```

Reply:
Fiz o merge por aqui :D